Source/WTF/wtf/WeakHashSet.h

```diff
@@ -116,9 +118,11 @@ class WeakHashSet final {
     bool contains(const U& value) const
     {
         auto& weakPtrImpl = value.weakPtrFactory().m_impl;
-        if (!weakPtrImpl || !*weakPtrImpl)
-            return false;
-        return m_set.contains(*weakPtrImpl);
+        if (auto* pointer = weakPtrImpl.pointer()) {
```

Comment:
Why not this?
auto* pointer = weakPtrImpl.pointer(); pointer && *pointer

Reply:
Changed.